api/app/controllers/review.controller.ts

```diff
@@ -16,6 +16,29 @@ import {
   buildIsInActiveCondition,
   buildIsPendingCondition,
 } from "../utils/review.inference";
+import { parseMdToJSON } from "../helpers/transcript";
+import axios from "axios";
+import { BaseParsedMdContent, TranscriptAttributes } from "../types/transcript";
+
+
+// THis function fetches and parses a transcript from a URL (already saved in the db which points to transcript on github), or returns the original transcript if no URL is provided. This is use to sync a transcript in review with the FE.
+const transcriptWrapper = async (transcript: TranscriptAttributes) => {
+  if (!transcript.transcriptUrl) return transcript;
+  try {
+    const response = await axios.get(transcript.transcriptUrl);
+    const transcriptData = parseMdToJSON<BaseParsedMdContent>(response.data);
+    const newTranscript = {
+      ...transcriptData, content: transcriptData.content.body
```

Comment:
`transcriptData.content.body` is not part of what the parser returns. The data you are looking for are in `transcriptData.body`.